shell/list/harvesterhci.io.management.cluster.vue

```diff
@@ -17,20 +18,32 @@ export default {
     TypeDescription,
     Loading
   },
-
+  mixins:     [ResourceFetch],
   props:      {
     schema: {
       type:     Object,
       required: true,
     },
+    loadResources: {
+      type:    Array,
+      default: () => []
+    },
+
+    loadIndeterminate: {
+      type:    Boolean,
+      default: false
+    },
+
+    incrementalLoadingIndicator: {
+      type:    Boolean,
+      default: false
+    },
   },
 
   async fetch() {
-    const inStore = this.$store.getters['currentProduct'].inStore;
-
     const hash = await allHash({
-      hciClusters:  this.$store.dispatch(`${ inStore }/findAll`, { type: HCI.CLUSTER }),
-      mgmtClusters: this.$store.dispatch(`${ inStore }/findAll`, { type: MANAGEMENT.CLUSTER })
+      hciClusters:  this.$fetchType(CAPI.RANCHER_CLUSTER, [CAPI.RANCHER_CLUSTER, MANAGEMENT.CLUSTER]),
```

Comment:
on `config/product/manager.js` we have:

```
configureType(CAPI.RANCHER_CLUSTER, {
    showListMasthead: false, namespaced: false, alias: [HCI.CLUSTER]
  });
```
  
I don't think alias are taken into account on incremental loading. The only relevant reference I could find was on `subscribe.js`. You are right about this. We are classifying it wrong because I just bypassed the `HCI.CLUSTER` and went straight for the `CAPI.RANCHER_CLUSTER` which the correct request, but employs the wrong classification.

To be discussed.

Reply:
reverted as it's a spoofed type